content/_layouts/frame.html.haml

```diff
@@ -25,8 +25,8 @@
     %meta{:content => "@JenkinsCI", :name => "twitter:creator"}/
 
     / Twitter Summary card images must be at least 120x120px
-    %meta{:content => "img/blueocean.gif", :name => "twitter:image"}/
-    %meta{:content => "img/blueocean.gif", :name => "twitter:image:src"}/
+    %meta{:content => expand_link(page.opengraph[:image]), :name => "twitter:image"}/
```

Comment:
The reason the build is failing is because on many pages `page.opengraph` doesn't exist. This should be under the `unless` statement on line 37ish